pkgs/development/python-modules/pacup/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  poetry2nix,
+  python,
+  fetchFromGitHub,
+}:
+poetry2nix.mkPoetryApplication rec {
+  pname = "pacup";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "pacstall";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Hl/Gq/cZz4RGYKTuyDlrhATAUYEzKEuYIm0JdToN/ZY=";
+  };
+
+  projectDir = src;
+
+  # Propagating dependencies leads to issues downstream
+  # We've already patched poetry to prefer "vendored" dependencies
+  postFixup = ''
+    rm -r $out/nix-support
+  '';
+
+  # Fails because of impurities (network, git etc etc)
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Help maintainers update pacscripts";
+    longDescription = "Pacup (Pacscript Updater) is a maintainer helper tool to help maintainers update their pacscripts. It semi-automates the tedious task of updating pacscripts, and aims to make it a fun process for the maintainer!";
```

This is rather long for description

---

It's the longDescription

---

oh, yeah right. It should still use a multiline string

```suggestion
    longDescription = ''
      Pacup (Pacscript Updater) is a maintainer helper tool to help maintainers update their pacscripts.
      It semi-automates the tedious task of updating pacscripts, and aims to make it a fun process for the maintainer!
    '';
```